### PR TITLE
[WIP] Add CSS modules

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Add CSS modules.

#### How should this be manually tested?
[Click here to access the workspace.](https://vitoria--storecomponents.myvtex.com/)

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/LoginOptions.js [hunk 1]
```diff
@@ -52,16 +54,16 @@ class LoginOptions extends Component {
 
     const { loadingOptions } = this.state
 
-    const classes = classNames('vtex-login-options', className, {
+    const classes = classNames([`${login.options}`], className, {
```
Comment (Klynger) [inline react/components/LoginOptions.js]: You don't need an array here

File: react/components/OAuth.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class OAuth extends Component {
               {children}
               <span className="t-action--small vtex-login__oauth-label relative normal">
                 {translate('loginOptions.oAuth', intl)}
-                <span className="vtex-login__oauth-provider b">{provider}</span>
+                <span className="b ml2">{provider}</span>
```
Comment (Klynger) [inline react/components/OAuth.js]: No token here ? :thinking:

File: react/components/RecoveryPassword.js [hunk 3]
```diff
@@ -78,12 +80,12 @@ class RecoveryPassword extends Component {
 
     return (
       <Form
-        className="vtex-login__email-verification"
+        className={`${login.emailVerification}`}
```
Comment (Klynger) [inline react/components/RecoveryPassword.js]: It should be just `className={login.emailVerification}`

File: react/components/RecoveryPassword.js [hunk 4]
```diff
@@ -142,13 +144,13 @@ class RecoveryPassword extends Component {
         }
         footer={
           <Fragment>
-            <div className="vtex-login__back-button">
+            <div className={`${login.backButton}`}>
```
Comment (Klynger) [inline react/components/RecoveryPassword.js]: No need for string interpolation

File: react/components/Tooltip.js [hunk 5]
```diff
@@ -2,11 +2,13 @@ import React, { Component } from 'react'
...
 import classNames from 'classnames'
 
+import login from '../login.css'
+
 export default class Tooltip extends Component {
   render() {
     const { children, title, top } = this.props
 
-    const classes = classNames('vtex-tooltip__container pa5 br2 absolute z-max', {
+    const classes = classNames([`${login.tooltipContainer} pa5 br2 absolute z-max bg-near-black`], {
```
Comment (Klynger) [inline react/components/Tooltip.js]: No need for this array

File: react/LoginContent.js [hunk 6]
```diff
@@ -289,16 +289,16 @@ class LoginContent extends Component {
       this.shouldRenderLoginOptions
     )
 
-    const className = classNames('vtex-login-content flex relative bg-base justify-around overflow-hidden', {
-      'vtex-login-content--initial-screen': this.state.isOnInitialScreen,
-      'vtex-login-content--always-with-options flex-column-reverse items-center flex-row-ns items-baseline-ns':
+    const className = classNames([`${login.content} center flex relative bg-base justify-around overflow-hidden pa4`], {
```
Comment (Klynger) [inline react/LoginContent.js]: You don't need to use this array

File: react/LoginContent.js [hunk 7]
```diff
@@ -289,16 +289,16 @@ class LoginContent extends Component {
...
+      [`${login.contentInitialScreen}`]: this.state.isOnInitialScreen,
+      [`${login.contentAlwaysWithOptions} mw6-ns flex-column-reverse items-center flex-row-ns items-baseline-ns`]:
         !isInitialScreenOptionOnly,
       'items-baseline': isInitialScreenOptionOnly,
     })
 
-    const formClassName = classNames('vtex-login-content__form dn ph4', {
-      [`vtex-login-content__form--step-${step}`]: step >= 0,
-      'vtex-login-content__form--visible db': this.shouldRenderForm,
+    const formClassName = classNames([`${login.contentForm} dn ph4`], {
```
Comment (Klynger) [inline react/LoginContent.js]: You don't need to use this array

File: react/components/CodeConfirmation.js [hunk 8]
```diff
@@ -76,13 +78,13 @@ class CodeConfirmation extends Component {
         }
         footer={
           <Fragment>
-            <div className="vtex-login__back-button">
+            <div className={`${login.backButton}`}>
```
Comment (Klynger) [inline react/components/CodeConfirmation.js]: You don't need to use string interpolation here

File: react/components/EmailAndPassword.js [hunk 9]
```diff
@@ -101,12 +103,12 @@ class EmailAndPassword extends Component {
 
     return (
       <Form
-        className="vtex-login__email-verification"
+        className={`${login.emailVerification}`}
```
Comment (Klynger) [inline react/components/EmailAndPassword.js]: You don't need to use string interpolation here

File: react/components/EmailAndPassword.js [hunk 10]
```diff
@@ -170,15 +172,15 @@ class EmailAndPassword extends Component {
         footer={
           <Fragment>
             {showBackButton && (
-              <div className="vtex-login__back-button">
+              <div className={`${login.backButton}`}>
```
Comment (Klynger) [inline react/components/EmailAndPassword.js]: You don't need to use string interpolation here

File: react/components/EmailVerification.js [hunk 11]
```diff
@@ -62,12 +64,12 @@ class EmailVerification extends Component {
 
     return (
       <Form
-        className="vtex-login__email-verification"
+        className={`${login.emailVerification}`}
```
Comment (Klynger) [inline react/components/EmailVerification.js]: You don't need to use string interpolation here

File: react/components/EmailVerification.js [hunk 12]
```diff
@@ -94,7 +96,7 @@ class EmailVerification extends Component {
         footer={
           <Fragment>
             {(showBackButton || isCreatePassword) && (
-              <div className="vtex-login__back-button">
+              <div className={`${login.backButton}`}>
```
Comment (Klynger) [inline react/components/EmailVerification.js]: You don't need to use string interpolation here

File: react/components/LoginOptions.js (hunk 1 above)
Comment (JuanAlmeida12) [reply react/components/LoginOptions.js]: You can remove array and interpolation string in this case `classNames(login.options,...`

File: react/components/LoginComponent.js [hunk 13]
```diff
@@ -86,18 +88,18 @@ export default class LoginComponent extends Component {
...
     return (
-      <div className="vtex-login__container flex items-center f6 fr">
+      <div className={`${login.container} flex items-center f6 fr`}>
         <div className="relative">
           {this.renderIcon()}
           <OutsideClickHandler onOutsideClick={onOutSideBoxClick}>
             <div
-              className={`vtex-login__box absolute z-max ${
+              className={`${login.box} absolute z-max ${
                 isBoxOpen ? 'flex' : 'dn'
```
Comment (Klynger) [inline react/components/LoginComponent.js]: You should use `classNames`

File: react/components/LoginOptions.js [hunk 14]
```diff
@@ -52,16 +54,16 @@ class LoginOptions extends Component {
 
     const { loadingOptions } = this.state
 
-    const classes = classNames('vtex-login-options', className, {
+    const classes = classNames(login.options, className, {
       'vtex-login-options--sticky': isAlwaysShown,
```
Comment (theomoura) [inline react/components/LoginOptions.js]: You could replace `vtex-login-options--sticky` for CSS Modules

File: react/components/LoginOptions.js [hunk 15]
```diff
@@ -52,16 +54,16 @@ class LoginOptions extends Component {
...
 
     return (
       <div className={classes}>
         <FormTitle>{title || translate(fallbackTitle, intl)}</FormTitle>
-        <ul className="vtex-login-options__list list pa0">
+        <ul className={`${login.optionsList} list pa0`}>
           {this.showOption('accessKeyAuthentication', 'loginOptions.emailVerification') &&
-            <li className="vtex-login-options__list-item mb3">
+            <li className={`${login.optionsListItem} mb3`}>
               <div className="vtex-login__button">
```
Comment (theomoura) [inline react/components/LoginOptions.js]: Same with `vtex-login__button`

File: react/components/LoginOptions.js [hunk 16]
```diff
@@ -73,7 +75,7 @@ class LoginOptions extends Component {
             </li>
           }
           {this.showOption('classicAuthentication', 'loginOptions.emailAndPassword') &&
-            <li className="vtex-login-options__list-item mb3">
+            <li className={`${login.optionsListItem} mb3`}>
               <div className="vtex-login__button">
```
Comment (theomoura) [inline react/components/LoginOptions.js]: Same here

File: react/components/LoginOptions.js [hunk 17]
```diff
@@ -99,7 +101,7 @@ class LoginOptions extends Component {
             )
           })}
           {!options && (
-            <li className="vtex-login-options__list-item mb3">
+            <li className={`${login.optionsListItem} mb3`}>
               <div className="vtex-login__button vtex-login__button--danger">
```
Comment (theomoura) [inline react/components/LoginOptions.js]: Same here

**Comment by brunojdo** [comment]
@vtex-apps/identity-xp will deal with this issue.

**Pull request closed without merging**